src/Language/PureScript/CST/Parser.y

```diff
@@ -582,30 +582,53 @@ binder :: { Binder () }
   | binder1 '::' type { BinderTyped () $1 $2 $3 }
```

Comment:
There's no need to generate this file and include it in the repo. 

Iirc, `cabal` does some magic builtin wizardry to compile the `happy` file and convert it into an `.hs` file and put it somewhere in the `dist-newstyle` directory for us.